[2025-01-03T21:42:50.354-0800] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-01-03T21:42:50.368-0800] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: CovidE2EPipeDag.clean_up_sql_exe scheduled__2025-01-01T00:35:00+00:00 [queued]>
[2025-01-03T21:42:50.377-0800] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: CovidE2EPipeDag.clean_up_sql_exe scheduled__2025-01-01T00:35:00+00:00 [queued]>
[2025-01-03T21:42:50.378-0800] {taskinstance.py:2866} INFO - Starting attempt 1 of 2
[2025-01-03T21:42:50.396-0800] {taskinstance.py:2889} INFO - Executing <Task(BashOperator): clean_up_sql_exe> on 2025-01-01 00:35:00+00:00
[2025-01-03T21:42:50.400-0800] {standard_task_runner.py:72} INFO - Started process 70749 to run task
[2025-01-03T21:42:50.408-0800] {standard_task_runner.py:104} INFO - Running: ['airflow', 'tasks', 'run', 'CovidE2EPipeDag', 'clean_up_sql_exe', 'scheduled__2025-01-01T00:35:00+00:00', '--job-id', '606', '--raw', '--subdir', 'DAGS_FOLDER/CovidE2EPipeDag.py', '--cfg-path', '/var/folders/vm/rn9v1g1j739_mq8j5pfl1yxm0000gn/T/tmp66ljqvb_']
[2025-01-03T21:42:50.411-0800] {standard_task_runner.py:105} INFO - Job 606: Subtask clean_up_sql_exe
[2025-01-03T21:42:50.475-0800] {task_command.py:467} INFO - Running <TaskInstance: CovidE2EPipeDag.clean_up_sql_exe scheduled__2025-01-01T00:35:00+00:00 [running]> on host Matthews-MacBook-Air.local
[2025-01-03T21:42:50.562-0800] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Matthew Horn' AIRFLOW_CTX_DAG_ID='CovidE2EPipeDag' AIRFLOW_CTX_TASK_ID='clean_up_sql_exe' AIRFLOW_CTX_EXECUTION_DATE='2025-01-01T00:35:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-01-01T00:35:00+00:00'
[2025-01-03T21:42:50.563-0800] {logging_mixin.py:190} INFO - Task instance is in running state
[2025-01-03T21:42:50.565-0800] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2025-01-03T21:42:50.566-0800] {logging_mixin.py:190} INFO - Current task name:clean_up_sql_exe state:running start_date:2025-01-04 05:42:50.368737+00:00
[2025-01-03T21:42:50.567-0800] {logging_mixin.py:190} INFO - Dag name:CovidE2EPipeDag and current dag run status:running
[2025-01-03T21:42:50.568-0800] {taskinstance.py:731} INFO - ::endgroup::
[2025-01-03T21:42:50.569-0800] {subprocess.py:78} INFO - Tmp dir root location: /var/folders/vm/rn9v1g1j739_mq8j5pfl1yxm0000gn/T
[2025-01-03T21:42:50.571-0800] {subprocess.py:88} INFO - Running command: ['/bin/bash', '-c', 'snow stage remove @raw_data_stage/load_staged_data_2025-01-03.sql;                   rm ~/airflow/helper_scripts/load_staged_data_2025-01-03.sql']
[2025-01-03T21:42:50.581-0800] {subprocess.py:99} INFO - Output:
[2025-01-03T21:42:55.009-0800] {subprocess.py:106} INFO - ╭─ Error ───────────────────────────────────╮
[2025-01-03T21:42:55.010-0800] {subprocess.py:106} INFO - │ Specified name                            │
[2025-01-03T21:42:55.011-0800] {subprocess.py:106} INFO - │ '@raw_data_stage/load_staged_data_2025-01 │
[2025-01-03T21:42:55.013-0800] {subprocess.py:106} INFO - │ -03.sql' is not valid name.               │
[2025-01-03T21:42:55.014-0800] {subprocess.py:106} INFO - ╰───────────────────────────────────────────╯
[2025-01-03T21:42:55.309-0800] {subprocess.py:110} INFO - Command exited with return code 0
[2025-01-03T21:42:55.337-0800] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-01-03T21:42:55.339-0800] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=CovidE2EPipeDag, task_id=clean_up_sql_exe, run_id=scheduled__2025-01-01T00:35:00+00:00, execution_date=20250101T003500, start_date=20250104T054250, end_date=20250104T054255
[2025-01-03T21:42:55.349-0800] {logging_mixin.py:190} INFO - Task instance in success state
[2025-01-03T21:42:55.350-0800] {logging_mixin.py:190} INFO -  Previous state of the Task instance: running
[2025-01-03T21:42:55.352-0800] {logging_mixin.py:190} INFO - Dag name:CovidE2EPipeDag queued_at:2025-01-04 05:39:56.542371+00:00
[2025-01-03T21:42:55.353-0800] {logging_mixin.py:190} INFO - Task hostname:Matthews-MacBook-Air.local operator:BashOperator
[2025-01-03T21:42:55.376-0800] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-01-03T21:42:55.411-0800] {standard_task_runner.py:217} INFO - Process not found (most likely exited), stop collecting metrics
[2025-01-03T21:42:55.413-0800] {taskinstance.py:3895} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-01-03T21:42:55.414-0800] {local_task_job_runner.py:245} INFO - ::endgroup::
